src/wordlist.c

```diff
@@ -574,6 +575,15 @@ void do_wordlist_crack(struct db_main *db, const char *name, int rules)
 
 		file_is_fifo = ((st.st_mode & S_IFMT) == S_IFIFO);
 
+#if OS_FORK
+		if (options.fork && file_is_fifo) {
+			if (john_main_process)
+				fprintf(stderr, "Error, cannot use --fork "
+					"with FIFO as dictionary file.\n");
```

Comment:
We normally say "wordlist", not "dictionary", and no need to wrap source code lines shorter than 132.

Reply:
Oops, I forgot that I wrapped lines in `wordlist.c`.

I replicated 'dictionary' from other warning. Now both places are fixed.